app/models/bishop.rb

```diff
@@ -1,13 +1,17 @@
-# This is code for how the Bishop should move
+legal# This is code for how the Bishop should move
```

Comment:
I do this all the time.  I end up with 'git pull' or other such typing in my code because I think I'm in the terminal.